### DISCUSSION TITLE
How to fetch updates and refresh statically rendered pages at runtime

### DISCUSSION THREAD

**Discussion by AshokChava** [vercel/next.js#12589] [Help] [answered]
We are using getStaticProps and statically rendering few paths some on run time as we are using fallback:true. This works perfectly for any new items those gets requires by the user and they get statically rendered and next requests are servers from these files instead of server call. 

But when we update a CMS asset, those changes need to be reflected on the statically rendered pages, we are capturing the CMS changes using webhooks and hoping to regenerate the files again or at least delete the existing copies in .next/server/<id>/pages of the static rendered files so that next request will regenerate them when the next request comes. 

When we delete the json and HTML files from .next folder they are getting server out of the in-memory. Is there a way disable in memory and regenerate the files.

Please let us know.

**Comment by timneutkens** [comment, marked as answer]
This RFC should help with your case: https://github.com/zeit/next.js/discussions/11552
